rules/windows/file/file_event/file_event_win_create_non_existent_dlls.yml

```diff
@@ -25,11 +26,12 @@ logsource:
     category: file_event
 detection:
     selection:
-        TargetFilename:
+        - TargetFilename:
             - 'C:\Windows\System32\WLBSCTRL.dll'
             - 'C:\Windows\System32\TSMSISrv.dll'
             - 'C:\Windows\System32\TSVIPSrv.dll'
             - 'C:\Windows\System32\wow64log.dll'
+        - TargetFilename|endswith: '\SprintCSP.dll'
```

Comment:
Need to update the title "Creation Of Non-Existent DLLs In System Folders"
AS "SprintCSP.dll" is  placed into a writable folder contained in the SYSTEM %PATH%

Reply:
Done